### PR TITLE
Get Visual Studio code coverage working

### PR DESCRIPTION
**Pull request by davkean**
This makes Code Coverage work in Visual Studio again, also updated code coverage settings to match CodeCov.

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -10,6 +10,13 @@
   <Import Project="$(RepoToolsetDir)Settings.props" Condition="'$(RepoToolsetDir)' != ''" />
   <Import Project="..\build\VisualStudio.XamlRules.props" />
 
+  <PropertyGroup>
+    <_IsVisualStudioDeveloperBuild Condition="'$(OfficialBuild)' != 'true' AND $(CIBuild) != 'true' AND '$(BuildingInsideVisualStudio)' == 'true'">true</_IsVisualStudioDeveloperBuild>
```
Comment (sharwell) [inline src/Directory.Build.props]: :grey_question: was `$(CIBuild)` supposed to be in single quotes?

**Comment by jmarolf** [comment]
@davkean change the branch to master so we'd run tests against this (thanks for using `git merge-base`).

As an aside, do we want to turn on ci for the language features branch [here](https://github.com/dotnet/dotnet-ci/blob/a7709bfdcc3b9bf463fd8bad550627d8b502e4b0/data/repolist.txt#L165-L169)?

**Comment by jmarolf** [comment]
@davkean this won't work because Apex only support MSTest at the moment and repotoolset only supports running xUnit tests.  I'll submit an update to you branch so this works.

**Comment by jmarolf** [comment]
@davkean the other branch you compared against in `git merge-base` does not have support for integration tests.  you need to base this branch off of the latest master to be able to add support.

**Comment by davkean** [comment]
I think you are thinking this is something different than what is it. This lets you run unit tests inside Visual Studio and see Code Coverage results in the UI itself. This isn't enabling coverage for integration tests. I need it in language services as I'm writing a ton of tests.

**Comment by jmarolf** [comment]
Whoops, sorry for messing with your PR @davkean

**Comment by jmarolf** [comment]
My Comment about CI still stands though, do you want me to turn it on for the language service branch?

**Comment by davkean** [comment]
If you'd like - I've been okay so far without it, as I merge into master every couple of days but if you'd like that confidence I'll set one up.

**Pull request merged**